.prettierrc

```diff
@@ -3,7 +3,7 @@
   "semi": true,
   "singleQuote": true,
   "trailingComma": "es5",
-  "printWidth": 80,
+  "printWidth": 70,
```

Comment:
just curios, what is the reason for that?

Reply:
for formatting line breakes